### DISCUSSION TITLE
Reusable Tooltip with FloatingArrow : get error when passing context

### DISCUSSION THREAD

**Discussion by huuthe0109** [floating-ui/floating-ui#2283] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/108788433/230632228-00f8e4c2-993d-4c23-a0b9-24d16bc02b7c.png)
@atomiks Tkanks you floating-ui
I try to implement the Reusable Tooltip but i got an error

I hope you can help me with that?

**Comment by atomiks** [comment, marked as answer]
`useTooltipContext()` itself has a `context` object (from `useFloating`, a nested context, separate from the component context), so in this case you can do:

```js
const {arrowRef, context: floatingContext, ...context} = useTooltipContext();

<FloatingArrow context={floatingContext} />
```

**Comment by huuthe0109** [comment]
Thanks for replying!,

I solved it
